src/mrpro/operators/IdentityOp.py

```diff
@@ -0,0 +1,34 @@
+"""Identity Operator."""
+
+import torch
+
+from mrpro.operators.LinearOperator import LinearOperator
+
+
+class IdentityOp(LinearOperator):
+    r"""The Identity Operator.
+
+    A Linear Operator that returns a single input unchanged.
+    """
+
+    def __init__(self) -> None:
+        """Initialize Identity Operator."""
+        super().__init__()
+
+    def forward(self, x: torch.Tensor) -> tuple[torch.Tensor]:
+        """Identity of input `x`.
```

Comment:
The ` before and after x will lead to x being printed slightly italic in the docu. Is this what you want here or did you want it to be a literal? 

see e.g. https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html